src/component/term/TermMetadataEdit.tsx

```diff
@@ -58,8 +61,15 @@ export class TermMetadataEdit extends React.Component<TermMetadataEditProps, Ter
     public onPrefLabelChange = (prefLabel : string) => {
         const update = {};
         update[this.props.language] = prefLabel;
-        this.setState({labelExists: false, label: Object.assign({}, this.state.label, update)});
-        if (prefLabel.toLowerCase() === getLocalized(this.props.term.label).toLowerCase()) {
+        const updateLabelExists = {};
+        updateLabelExists[this.props.language] = false;
+        this.setState({
+            label: Object.assign({}, this.state.label, update),
+            labelExist: Object.assign({}, this.state.labelExist, updateLabelExists)
```

Comment:
Why is the `labelExist` attribute initialized as Map, when it is on the first label edit replaced by a plain object?